assets/scss/02-molecules/_sticky-nav.scss

```diff
@@ -147,8 +147,8 @@
     }
   }
 }
-
-//theme
+
+//theme
 .ma__sticky-nav {
   @media ($bp-medium-max) {

```

Comment:
I do not understand why the `@` here is colored red in this PR. Is there a change here?

Reply:
I'm not sure why it does that either, actually.